src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,33 +1,66 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
-    # TO-DO
-    
+# TO-DO: complete the helper function below to merge 2 sorted arrays
+def merge(arrA, arrB):
+    merged_arr = []
+    i = 0
+    j = 0
+
+    while i < len(arrA) and j < len(arrB):
+        if arrA[i] < arrB[j]:
```

Comment:
Very nice! One of the most clear implementations of the merge function I've seen. Great work